### PR TITLE
fix: use body as string to parse it as JSON

### PR DESCRIPTION
**Pull request by dipak-pawar**
**Changes Proposed in this PR**
* Updates import path to use from `fabric8-cluster-client` before copying it to fabric8-cluster-client, so that it will compile successfully.
* Removes `/cluster` and `/tool` completely to get all new changes without any conflicts before copying new changes.
* Updates PR body description to add commits for changes in design package only.

### CODE REVIEW

File: .scripts/create_push_client.sh [hunk 1]
```diff
@@ -17,48 +17,49 @@ function generate_client_and_create_pr() {
...
     local body=$(pr_body)
     local short_head=$(git rev-parse --short HEAD)
-    local branch="client_update_${short_head}"
+    local branch="client_update_${short_head}_$(date +%s)"
 
     cd /tmp/${GHREPO}
     git checkout -b ${branch}
     cd -
+    for i in $(find tool -name "*.go"); do
+        sed -i 's:"github.com/fabric8-services/fabric8-cluster/cluster":"github.com/fabric8-services/fabric8-cluster-client/cluster":' "$i";
```
Comment (alexeykazakov) [inline .scripts/create_push_client.sh]: There should not be any repo specific code (fabric8-cluster-*) in common. The code in common should work for any repo or it should be moved to fabric8-cluster-client repo.

**Review by alexeykazakov — CHANGES_REQUESTED** [review]

File: .scripts/create_push_client.sh (hunk 1 above)
Comment (dipak-pawar) [reply .scripts/create_push_client.sh]: Addressed in a39b4f2

**Review by alexeykazakov — APPROVED** [review]

File: .scripts/create_push_client.sh [hunk 2]
```diff
@@ -17,54 +17,58 @@ function generate_client_and_create_pr() {
...
+    echo $description$commits
 }
 
 function generate_client_setup() {
     SERVICE_NAME=${PWD##*/}
-    GHORG=${1:-fabric8-services}
-    GHREPO=${2:-${SERVICE_NAME}-client}
+    PKG_NAME=${1}
+    TOOL_DIR=${2:-tool/cli}
+    GHORG=${3:-fabric8-services}
```
Comment (alexeykazakov) [inline .scripts/create_push_client.sh:71]: I think we need to add some comments here with descriptions of the params we expect here. And how they will be used.
GitHub Org and Repo are self-explanatory but new params are not.

**Review by alexeykazakov — APPROVED** [review]
Overall looks good. But need some docs/comments.

**Pull request merged**